src/js/core/methods/CardanoSignTransaction.js

```diff
@@ -116,17 +117,22 @@ export default class CardanoSignTransaction extends AbstractMethod {
             });
         }
 
+        let auxiliaryData;
+        if (payload.auxiliaryData) {
```

Comment:
An empty byte sequence is probably valid auxiliary data, so I'd rather explicitly check `payload.auxiliaryData != null`